src/pages/settings/Profile/PersonalDetails/DateOfBirthPage.js

```diff
@@ -93,6 +93,7 @@ class DateOfBirthPage extends Component {
                 >
                     <View>
                         <DatePicker
+                            placeholder="yyyy-mm-dd"
```

Comment:
Looking good, just want to get a hipcheck if this should be `YYYY-MM-DD` or what you got here. cc @Beamanator @shawnborton do we want to write the placeholder format uppercase or lowercase?

Reply:
Hmm I don't feel too strongly, I could go either way. @JmillsExpensify or @trjExpensify any strong feelings here?

Reply:
we can still with lower case for now - to be consistence with web/desktop (dd/mm/yyyy). And when we migrate to new date picker we can revisit and see which one to use camelcase or lowercase. wdyt guys

Reply:
I don't have any strong feelings on that one, though I don't recall typically seeing date format examples in lowercase, so if I had to choose I'd go with `YYYY-MM-DD`. Agree that it should be consistent on all platforms though.

Reply:
I see, yeah, although I also like uppercase more, I think being consistent with the lowercase is better. Lets stick with what you have here then